src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -1027,6 +1027,14 @@ public String getReleaseAnnotation(ReleaseLevel releaseLevel) {
     return getNotImplementedString("SurfaceNamer.getReleaseAnnotation");
   }
 
+  public String getLinkedElementName(ProtoElement element) {
```

Comment:
These are used outside of the `NodeJSSurfaceNamer`. I'll look into moving these methods to the JSCommentReformatter.